pages.es/linux/torify.md

```diff
@@ -0,0 +1,32 @@
+# torify
+
+> Enruta el tráfico de red a través de la red Tor.
+> Más información: <https://manned.org/man/torify>.
+
+- Enruta el tráfico a través de Tor:
+
+`torify {{comando}}`
+
+- Activa Tor en el intérprete de comandos:
+
+`torify {{on|off}}`
+
+- Crea un intérprete de órdenes habilitado para Tor:
+
+`torify --shell`
+
+- Comprueba si hay un intérprete de órdenes habilitado para Tor:
+
+`torify show`
+
+- Especifica el archivo de configuración de Tor:
+
+`torify -c {{archivo-configuración}} {{comando}}`
```

Comment:
```suggestion
`torify -c {{archivo_de_configuración}} {{comando}}`
```

The corresponding [placeholder in the original page](https://github.com/tldr-pages/tldr/blob/c476e22061eb604a0a5cf589a4b886ce041f0642/pages/linux/torify.md?plain=1#L24) should be updated as well.